pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
```

Comment:
Currently we didn't use package manager, plz remove `poetry` files.

Reply:
Yes my bad, this shouldn't have been part of the commit, I removed it